src/metabase/driver/crate.clj

```diff
@@ -58,6 +60,40 @@
 (defn- string-length-fn [field-key]
   (hsql/call :char_length field-key))
 
+(defn- describe-table-fields
+  [database, driver, {:keys [schema name]}]
+  (set (doseq [{column_name :column_name, type_name :type_name}
+             (jdbc/query
+                 (sql/db->jdbc-connection-spec database)
+                 [(format "select column_name, data_type as type_name
+                  from information_schema.columns
+                  where table_name like '%s' and table_schema like '%s'" name schema)])]
+          (merge {:name      column_name
+                 :custom    {:column-type type_name}
```

Comment:
Also it looks like that `merge` isn't merging any maps (it only has one arg) so it's not doing anything / unneeded